src/app/components/applications/SSV/RegisterValidatorHome/components/ImportFile/flows/KeyShares.tsx

```diff
@@ -241,9 +244,13 @@ const KeyShareFlow = () => {
       setValidatorsList(validators);
     };
 
-    const changeCountOfValidators = (e: any): void => {
-      let { value } = e.target;
-      value = Number(value) === 0 ? config.GLOBAL_VARIABLE.MIN_VALIDATORS_COUNT_PER_BULK_REGISTRATION : value;
+    const changeCountOfValidators = (value: number): void => {
+      if (Number(value) === 0) {
```

Comment:
Don't need to wrap value with number as it should be number according to the function signature.